backend/src/LibExecution/Interpreter.fs

```diff
@@ -59,19 +59,21 @@ let rec eval' (state : ExecutionState) (st : Symtable) (e : Expr) : DvalTask =
     | ENull _id -> return DNull
     | ECharacter (_id, s) -> return DChar s
 
-
-    | ELet (_id, lhs, rhs, body) ->
+    | ELet (_id, pat, rhs, body) ->
       let! rhs = eval state st rhs
+
       match rhs with
-      // CLEANUP we should still preview the body
```

Comment:
Use a simplified form of the code in #4711 for this (which _does_ preview the body)